cmd/client_changefeed.go

```diff
@@ -739,3 +742,19 @@ func getUpstreamCredential() *security.Credential {
 		KeyPath:  upstreamSslKeyPath,
 	}
 }
+
+func confirmChangefeedRemove(cmd *cobra.Command) error {
+	if noConfirm {
+		return nil
+	}
+	cmd.Printf("Confirm that you want to remove this changefeed [Y/N]\n")
+	var yOrN string
```

Comment:
see this code block several times, can we unify all the cases, thx?

Reply:
Sorry, I don't quite know what you mean.

Reply:
sorry, what i mean is the check `y/n` logic been used several times in this file, maybe could extract the same logic to a func instead.

Reply:
Ok, sure. It's a good suggestion.